README.md

```
@@ -18,25 +190,30 @@ For contributors: see [AGENTS.md](AGENTS.md)
   - [ ] MCP Prompts
   - [ ] MCP Resources
+- [ ] Host this app
+- [ ] Add info about [Open Inference](https://www.openinference.xyz/) into the settings dialog
+- [ ] Allow theme CSS variables for the component
 - [ ] One-click MCP install
 - [ ] Add your own MCP server
 - [x] MCP UI support
-- [ ] Local LLM support (via Transformers JS)
+- [ ] [Built-in AI support](https://ai-sdk.dev/providers/community-providers/built-in-ai) (Transformers JS + AI SDK + [built-in-ai](https://github.com/jakobhoeg/built-in-ai)) 
+  - Examples
+    - https://huggingface.co/spaces/ibm-granite/Granite-4.0-WebGPU/blob/main/src/worker.js
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)